eclipse-scout-core/src/filechooser/FileChooser.js

```diff
@@ -294,7 +294,7 @@ export default class FileChooser extends Widget {
       this.fileInput.validateMaximumUploadSize(files);
     } catch (errorMessage) {
       MessageBoxes.createOk(this)
-        .withHeader(this.session.text('ui.FileSizeLimitTitle'))
```

Comment:
Do you really want to change the header/title? If yes, is the text-key 'FileSizeLimitTitle' still used?

Maybe you'd better leave header/title untouched and add the result from getErrorMessageMaximumUploadSizeExceeded to the body/errorMessage?